packages/arb-token-bridge-ui/tests/e2e/specs/deposit.cy.ts

```diff
@@ -0,0 +1,142 @@
+  const goerliProvider = new JsonRpcProvider(rpcURL)
+  return await goerliProvider.getBalance(Cypress.env('ADDRESS'))
+}
+
+const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env('INFURA_KEY')}`
+const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
+
+/**
+ * When user wants to bridge ETH from L1 to L2
+ */
+describe('Deposit ETH', () => {
+  // Happy Path
+  context('user has some ETH and is on L1', () => {
+    let l1ETHbal
+    let l2ETHbal
+
+    before(() => {
+      cy.clearLocalStorageSnapshot()
+      getInitialETHBalance(goerliRPC).then(
+        val => (l1ETHbal = formatBigNumber(val, 18, 5))
+      )
+      getInitialETHBalance(arbitrumGoerliRPC).then(
+        val => (l2ETHbal = formatBigNumber(val, 18, 5))
+      )
+      cy.disconnectMetamaskWalletFromAllDapps()
+      cy.login()
+    })
+
+    beforeEach(() => {
+      // restore local storage from first test
+      cy.restoreLocalStorage()
+    })
+
+    afterEach(() => {
+      // cypress clears local storage between tests
+      // so in order to preserve local storage on the page between tests
+      // we need to use the cypress-localstorage-commands plugin
+      // or else we have to visit the page every test which will be much slower
+      // https://docs.cypress.io/api/commands/clearlocalstorage
+      cy.saveLocalStorage()
+    })
+
+    it('should show L1 and L2 chains correctly', () => {
+      cy.findByRole('button', { name: /From: Goerli/i }).should('be.visible')
+      cy.findByRole('button', { name: /To: Arbitrum Goerli/i }).should(
+        'be.visible'
+      )
+    })
+
+    it('should show L1 and L2 ETH balances correctly', () => {
+      cy.findByText(`Balance: ${l1ETHbal} ETH`).should('be.visible')
+      cy.findByText(`Balance: ${l2ETHbal} ETH`).should('be.visible')
+    })
+
+    it('should show empty bridging summary', () => {
+      cy.findByText('Bridging summary will appear here.').should('be.visible')
+    })
+
+    context("bridge amount is lower than user's L1 ETH balance value", () => {
+      const zeroToLessThanOneETH = /0(\.\d+)*( ETH)/
+      it('should show summary', () => {
+        cy.findByPlaceholderText('Enter amount')
+          // https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Scrolling
+          // cypress by default tries to scroll the element into view even when it is already in view
+          // for unknown reasons, probably due to our root div's overflow:hidden CSS property,
+          // cypress would wrongly scroll the div and bring the element to the top of the view
+          // and in turn include the full moon into the view, cropping the header out of visible area
+          // to circumvent this bug with cypress, scrollBehaviour should be set false for this element
+          // because the element is already in view and does not require scrolling
+          // https://github.com/cypress-io/cypress/issues/23898
+          .type('0.0001', { scrollBehavior: false })
```

Comment:
`{ scrollBehavior: false }` is a temporary workaround
Please refer to https://github.com/cypress-io/cypress/issues/23898